docs/Readme.md

```diff
@@ -1,12 +1,15 @@
 # SynthDet Documentation
+<img src="images/Synthetic Data pipeline-Perception Workflow.png" align="middle"/>
+
 ## Installation & Setup
 * [Prerequisites](Prerequisites.md)
 
-## Getting Started 
-* [Getting Started with SynthDet](GettingStartedSynthDet.md)
-* [Running SynthDet in Unity Simulation](RunningSynthDetCloud.md)
+## Getting Started with data generation in Unity
+* Data generation example project - [Getting Started with SynthDet](GettingStartedSynthDet.md)
+* Scaling up data generation - [Running SynthDet in Unity Simulation](RunningSynthDetCloud.md)
+* Evaluating a dataset - [Dataset Evaluation with Dataset Insights framework](https://github.com/Unity-Technologies/datasetinsights/blob/master/docs/source/Evaluation_Tutorial.md)
```

Comment:
Should this be Dataset Insights or datasetinsights?

Reply:
Could go either way, but on https://github.com/Unity-Technologies/datasetinsights the title text says "Dataset Insights"

Reply:
https://datasetinsights.readthedocs.io/en/latest/Evaluation_Tutorial.html We should use this rendered version of official datasetinsights doc. It also includes documents for the datasetinsighs modules.

Reply:
Agreed, that was the next step, I just wanted to make sure we agreed on this sort of diagram first.  I already have the png for the DI page